includes/class-wppo-cli-command.php

```
@@ -0,0 +1,373 @@
+				WP_CLI::error( sprintf( __( 'Invalid cache action "%s". Did you mean "wp wppo cache clear"?', 'performance-optimisation' ), $action ) );
+				return;
+			}
+
+			require_once WPPO_PLUGIN_PATH . 'includes/class-log.php';
+			require_once WPPO_PLUGIN_PATH . 'includes/class-cache.php';
+
+			$page = $assoc_args['page'] ?? null;
+
+			if ( $page ) {
+				$path    = wp_normalize_path( trim( (string) wp_parse_url( $page, PHP_URL_PATH ), '/' ) );
+				$cleared = Cache::clear_cache( $path );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Cache' in method 'cache'. (undefined)

(StaticAccess)